docs/smart-contracts/install/local-dev-node.md

```diff
@@ -0,0 +1,247 @@
+
+1. Open or unlock Chainweaver.
+2. Click **Settings** in the Chainweaver navigation panel.
+3. Click **Network**.
+4. Under Edit Networks, type the network name **development**, then click **Create**.
+5. Expand the **development** network, then add the localhost as a node for this network by typing **127.0.0.1:8080**.
+
+   If the local computer is still running the development network Docker container, you should see the dot next to the node turn green.
+
+6. Click **Ok** to close the network settings.
+
+   After you click Ok, you can see **development** selected as your current network.
+   All actions you perform in Chainweaver are now executed on the local development network.
+
+## Install a standalone node
+
+If you want full control over node operations, you might want to install and manage `chainweb-node` as a standalone local node for development or as a node participating in the Kadena test or production public network.
+
+There are several options for setting up a Chainweb node in a physical or virtual environment.
+For example, you can install and run node binaries:
+
+- Directly on a physical Linux or macOS computer.
+- On an Linux or macOS instance with infrastructure from a cloud services provider.
+- From a Linux or macOS image in a Docker container.
+
+You can also build the binaries directly from the source code.
+
+### Minimum system requirements
+
+Before installing Chainweb, you should verify that your computer meets the following minimum recommended hardware requirements for running a node:
+
+- CPU: Minimum of two (2) CPU cores.
+- RAM: Minimum of four (4) GB of RAM.
+- Storage: Minimum 250 GB using a solid state drive (SSD) or fast hard disk drive (HDD).
+- Network: Publicly-accessible IP address.
+- Operating system: Linux AMD64 architecture.
+
+If you also plan to use the node for mining, to submit transactions, or to query blockchain data, you should verify that your computer meets the following minimum recommended hardware requirements: 
+
+- Four (4) CPU cores.
+- Eight (8) GB of RAM.
+
+#### Operating system architecture
+
+Most `chainweb-node` software runs on Linux or macOS with AMD64 architectures.
+If you want to run a Chainweb node on Linux or macOS with ARM64 architecture, you'll need to build the binaries yourself from the source code.
+You might experience some unexpected behavior when running a node on ARM64 architecture.
+Note that you should not configure **mining** for nodes on ARM64 architecture.
+RUnning a mining node on ARM64 architecture could cause your node to fork from the network.
+
+If you want to run a Chainweb node on Microsoft Windows, you should note that this isn't a supported configuration and requires you to install the latest Windows Services for Linux (WSL) software.
+If you choose to proceed after installing WSL, you can install the `chainweb-node` software in the WSL virtual environment rather than using the native Windows operating system. 
```

Comment:
Is this the case for running Chainweb? I thought the WSL instructions are only for running a pact binary and chainweb always relies on docker (I might be wrong). @chessai